src/test/java/teammates/ui/webapi/SearchStudentsNoServiceActionTest.java

```diff
@@ -0,0 +1,71 @@
+package teammates.ui.webapi;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.util.Const;
+import teammates.test.TestProperties;
+import teammates.ui.output.MessageOutput;
+
+/**
+ * SUT:{@link SearchStudentsAction}.
+ */
+public class SearchStudentsNoServiceActionTest extends BaseActionTest<SearchStudentsAction> {
```

Comment:
The no-service tests have been merged with the normal test counterparts, so none of these `*NoService*Test` are necessary anymore

Reply:
Removed `SearchStudentsNoServiceActionTest`, `SearchInstructorsNoServiceActionTest` and `NoServiceSearchTest`